### ISSUE TITLE
Cannot Publish Card Review

### ISSUE THREAD

**Issue by Agent-Chezni** [Null-Signal-Games/netrunnerdb#252]
Hello and thanks for all you do!

I got a bit inspired and wrote an 8,000 character review on Awakening Center...

I have it saved locally, but when I go to post it, I get the error "An error occurred while posting your review (OK). Reload the page and try again."

Is it being rejected b/c of the character count, or is there something else?

I've attached a .txt file of the text in case it helps.
Thanks!
--Chez

[Awakening Center.txt](https://github.com/Alsciende/netrunnerdb/files/2574798/Awakening.Center.txt)

**Comment by NoahTheDuke** [comment]
Holy shit lol, that’s amazing. Let me check, I do believe there is a character limit but I don’t remember off the top of my head.

**Comment by Saintis** [comment]
I believe a limit was put in when someone posted a decklist with the full text of Moby Dick in the description...

**Comment by x3N1GM4x** [comment]
I tried to reproduce the issue on my local instance, but it posted the review without issue. I'm reluctant to test in production right now as I don't think there's a front-end mechanism for removing a review (at least not for non-admins like me).

It looks like the full error is written to the console, lines 231-234 of `zoom.js`:

```
error: function (jqXHR, textStatus, errorThrown) {
    console.log('[' + moment().format('YYYY-MM-DD HH:mm:ss') + '] Error on ' + this.url, textStatus, errorThrown);
    form.replaceWith('<div class="alert alert-danger" role="alert">An error occured while posting your review (' + jqXHR.statusText + '). Reload the page and try again.</div>');
}
```

@Agent-Chezni - can you confirm that you still get this issue if you try to post the review now and if so, can you check the console within your browser for any message of the format `[YYYY-MM-DD HH:mm:ss] Error on {something}` ?

**Comment by ghost** [comment]
I got this too and wrote an articel about 3x times. Had to login explicitly to avoid it from happening.

**Comment by x3N1GM4x** [comment]
Closing as should be resolved by #285.
@Agent-Chezni  - let us know if you're still having this problem.
